CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 * [Fix] Fix Twice message printing when switching to the current connection ([#772](https://github.com/ploomber/jupysql/issues/772))
 * [Fix] Error when using %sqlplot in snowflake ([#697](https://github.com/ploomber/jupysql/issues/697))
 * [Doc] Fixes documentation inaccuracy that said `:variable` was deprecated (we brought it back in `0.9.0`)
+* [Feature] Added clickhouse integration tests
```

Comment:
this internal changes do not go into the changelog. we only put things in the changelog if they affect the end user, I added the "no-changelog" label to this PR so the tests pass even if there are no changes to the changelog